go/embedded/online_features.go

```diff
@@ -117,7 +117,7 @@ func (s *OnlineFeatureService) GetEntityTypesMapByFeatureService(featureServiceN
 		}
 		for _, entityName := range view.Entities {
```

Comment:
we can replace this loop to `range view.EntityColumns` since it already has join key and type, right?